test/test_torch.py

```diff
@@ -9787,6 +9787,24 @@ def run_subtest(guess_rank, actual_rank, matrix_size, batches, device, pca, **op
         guess_rank, actual_rank, size, batches = 2, 2, (17, 4), ()
         run_subtest(guess_rank, actual_rank, size, batches, device, jitted)
 
+    @unittest.skipIf(not TEST_NUMPY, "Numpy not found")
+    def test_ldexp(self, device):
```

Comment:
If the current implementation is used, add a test for the error cases, too. That is, add test cases that hit the various TORCH_CHECKs.